tests/ttnn/profiling/ops_for_profiling.py

```diff
@@ -584,7 +584,7 @@ def unary_div_bw(x, y):
         "op": ttnn.add,
         "name": "ttnn.complex_add",
         "is_complex": [True, True],
-        "need_out_mem_cfg": True,
+        # "need_out_mem_cfg": True,
```

Comment:
remove?

Reply:
Removed

Reply:
@ayerofieiev-tt Can you checkout if pull req is ok now?